cmd/eksctl-anywhere/cmd/generatehardware.go

```diff
@@ -50,6 +51,7 @@ var generateHardwareCmd = &cobra.Command{
 func init() {
 	generateCmd.AddCommand(generateHardwareCmd)
 	generateHardwareCmd.Flags().StringVarP(&hOpts.csvPath, "filename", "f", "", "path to csv file")
+	generateHardwareCmd.Flags().StringVarP(&hOpts.outputPath, "output", "o", "", "directory path to output hardware files")
```

Comment:
Added option to export hardware bundle to the directory provided. This is needed to export the hardware.yaml to cluster config directory during test run.

Reply:
Also, I think this is a nice to have option as customer may desire to export to another directory as well

Reply:
Should we go the full mile and let them specify the full file path (i.e. including file name)? Feels like the flexibility we're going for here.